Description: help autotools to find the correct sched dynamic shared library.
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -44,7 +44,7 @@
     gui_urls.xml
 
 AM_CXXFLAGS += $(MYSQL_CFLAGS)
-AM_LDFLAGS += -static
+AM_LDFLAGS += -static -L../lib/.libs -L../sched
 
 cancel_jobs_SOURCES = cancel_jobs.cpp
 cancel_jobs_LDADD = $(SERVERLIBS)
--- a/vda/Makefile.am
+++ b/vda/Makefile.am
@@ -4,7 +4,7 @@
 vda_PROGRAMS = vda vdad ssim
 
 AM_CXXFLAGS += $(MYSQL_CFLAGS)
-AM_LDFLAGS += -static
+AM_LDFLAGS += -static -L../lib/.libs -L../sched
 
 vda_SOURCES = vda.cpp vda_lib.cpp vda_lib2.cpp vda_policy.cpp stats.cpp
 vda_LDADD = $(SERVERLIBS)
--- a/sched/Makefile.am
+++ b/sched/Makefile.am
@@ -4,7 +4,7 @@
 include $(top_srcdir)/Makefile.incl
 
 AM_CPPFLAGS += $(MYSQL_CFLAGS) $(PTHREAD_CFLAGS)
-AM_LDFLAGS += -static
+AM_LDFLAGS += -static -L../lib/.libs/
 
 if ENABLE_LIBRARIES
 
--- a/apps/Makefile.am
+++ b/apps/Makefile.am
@@ -8,15 +8,15 @@
 
 upper_case_SOURCES = upper_case.cpp
 upper_case_CXXFLAGS = $(PTHREAD_CFLAGS)
-upper_case_LDFLAGS = -static-libtool-libs $(PTHREAD_CFLAGS)
+upper_case_LDFLAGS = -static-libtool-libs $(PTHREAD_CFLAGS) -L../api/.libs -L../lib/.libs
 upper_case_LDADD = $(APPLIBS)
 
 concat_SOURCES = concat.cpp
 concat_CXXFLAGS = $(PTHREAD_CFLAGS)
-concat_LDFLAGS = -static-libtool-libs $(PTHREAD_CFLAGS)
+concat_LDFLAGS = -static-libtool-libs $(PTHREAD_CFLAGS) -L../api/.libs -L../lib/.libs
 concat_LDADD = $(APPLIBS)
 
 1sec_SOURCES = 1sec.cpp
 1sec_CXXFLAGS = $(PTHREAD_CFLAGS)
-1sec_LDFLAGS = -static-libtool-libs $(PTHREAD_CFLAGS)
+1sec_LDFLAGS = -static-libtool-libs $(PTHREAD_CFLAGS) -L../api/.libs -L../lib/.libs
 1sec_LDADD = $(APPLIBS)
--- a/clientgui/Makefile.am
+++ b/clientgui/Makefile.am
@@ -6,6 +6,7 @@
 
 include $(top_srcdir)/Makefile.incl
 
+AM_LDFLAGS +=  -L../lib/.libs
 if ENABLE_CLIENT_RELEASE
   AM_LDFLAGS += -static-libtool-libs
 ## for an entirely statically linked library, you may want to try
